ironfish/src/wallet/index.ts

```diff
@@ -2,6 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 export * from './account/account'
+export { Format } from './account/encoder/encoder'
```

Comment:
Do we want this exported from `wallet/index.ts`? Seems like it would be better if we exported from `wallet/account/index.ts`

Reply:
Agreed, I'll add an index file at that level